src/libs/ui/mainwindow.cpp

```diff
@@ -218,6 +218,15 @@ MainWindow::MainWindow(Core::Application *app, QWidget *parent) :
     connect(ui->actionForward, &QAction::triggered, this, [this]() { currentTab()->forward(); });
     addAction(ui->actionForward);
 
+    shortcut = new QShortcut(QStringLiteral("Ctrl++"), this);
+    connect(shortcut, &QShortcut::activated, this, [this]() { currentTab()->zoomIn(); });
+    shortcut = new QShortcut(QStringLiteral("Ctrl+="), this);
```

Comment:
With the change suggested above, I am not sure we need to duplicate, but I haven't checked.

Reply:
It seems I didn't click Comment, anyway, I tested this and Ctrl+= is needed, ZoomIn only replaces Ctrl++